JobConfig/cosmic/cosmic_s1_dsstops.fcl

```diff
@@ -1,216 +1,15 @@
-# Mu2eG4 cuts
-
-crvStage1End:
-{
-   type: inVolume
-   pars: [DSOuterCryoShell, DSInnerCryoShell, DSFront, DSUpEndWallShell, TS5OuterCryoShell, TS5CryoInsVac, TS5Vacuum, Coll51, DS1Vacuum, DS2Vacuum, DS3Vacuum]
-   write: crvStage1
-}
-
-crvOuterBox:
-{
-   type: union
-   pars:
-   [
-      // Upstream plane
-      { type: plane normal: [  0, 0, -1 ] point : [ 0, 0, -3400 ] },
-
-      // Downstream plane
-      { type: plane normal: [  0, 0, 1 ] point : [ 0, 0, 19800 ] },
-
-      // +x plane
-      { type: plane normal: [  1, 0, 0 ] point : [ 1500, 0, 0 ] },
-
-      // -x plane (outside of cryo box)
-      { type: plane normal: [  -1, 0, 0 ] point : [ -8200, 0, 0 ] },
-
-      // +y plane
-      { type: plane normal: [  0, 1, 0 ] point : [ 0, 4000, 0 ] },
-
-      // -y plane
-      { type: plane normal: [  0, -1, 0 ] point : [ 0, -2100, 0 ] }
-   ]
-}
-
-crvInnerBox:
-{
-   type: intersection
-   pars:
-   [
-      // Upstream plane
-      { type: plane normal: [  0, 0, 1 ] point : [ 0, 0, -2200 ] },
-
-      // Downstream plane
-      { type: plane normal: [  0, 0, -1 ] point : [ 0, 0, 18500 ] },
-
-      // +x plane
-      { type: plane normal: [  -1, 0, 0 ] point : [ -1500, 0, 0 ] },
-
-      // -x plane
-      { type: plane normal: [  1, 0, 0 ] point : [ -6300, 0, 0 ] },
-
-      // +y plane
-      { type: plane normal: [  0, -1, 0 ] point : [ 0, 2500, 0 ] },
-
-      // -y plane
-      { type: plane normal: [  0, 1, 0 ] point : [ 0, -2100, 0 ] }
-   ]
-}
-
-cosmicKineticEnergyCuts:
-{
-   type: union
-   pars:
-   [
-      {
-        type: intersection
-        pars:
-        [
-          { type: union pars: [ @local::crvOuterBox, @local::crvInnerBox ] },
-          { type: kineticEnergy cut: 10.0 },
-          { type: pdgId pars: [ 22, 2112 ] }
-        ]
-      },
-      {
-      // 5 MeV can only make a hit in 1 CRV layer
-        type: intersection
-        pars:
-        [
-          { type: union pars: [ @local::crvOuterBox, @local::crvInnerBox ] },
-          { type: kineticEnergy cut: 5.0 },
-          { type: pdgId pars: [ 2212, 11, -11 ] }
-        ]
-      }
-   ]
-}
-#----------------------------------------------------------------
-END_PROLOG
-
-#================================================================
-# Give this job a name.
-process_name :  cosmics1
-
-source :
-{
-   module_type : EmptyEvent
-   maxEvents : @nil
-}
-
-services :
-{
-
-   message               : @local::default_message
-   TFileService          : { }
-   RandomNumberGenerator : {defaultEngineKind: "MixMaxRng" }
-
-   GeometryService        : { inputFile      : "JobConfig/cosmic/geom_cosmic_PhaseI.txt" }
-   ConditionsService      : { conditionsfile : "Mu2eG4/test/conditions_01.txt"      }
-   GlobalConstantsService : { inputFile      : "Mu2eG4/test/globalConstants_01.txt" }
-   G4Helper               : { }
-   SeedService            : @local::automaticSeeds
-}
+process_name :  CosmicS1
```

Comment:
Please rename the process 'primary'.  Otherwise this exposes the origin process in Ensemble files.

Reply:
Do you need the process name to be 'primary' for stage-1 as well?
I think stage-1 process name is lost after stage-2 job.
I'm sure I'm missing a point. 
I'll change the name to 'Primary' though.